test/test.js

```diff
@@ -9,35 +9,77 @@ let infura1155NFTInstance;
 contract('Infura721aNFTContract', function (accounts) {
 
     it("Contract deployment", function () {
-        //Fetching the contract instance of our smart contract
-        return Infura721aNFT.deployed().then(function (instance) {
+        return Infura721aNFT.deployed({from: accounts[0]}).then(function (instance) {
```

Comment:
is there a reason you don't just do `.deployed()`? it should be deploying from `accounts[0]`

Reply:
Fixed!